### ISSUE TITLE
Can't pass the casing check, because no capital letters in Hindi

### ISSUE THREAD

**Issue by msmolyakov** [sindresorhus/awesome-lint#100]
I am getting an error
```
List item description must start with valid casing
```
on this [link description](https://github.com/msmolyakov/awesome-waves/blob/1b55f04796bcaed81ab4df495222418d51d20cfb/README.md#hindi)
```markdown
- [Telegram](https://t.me/wavesdappsid) - सॉफ्टवेयर डेवलपर्स के लिए टेलीग्राम चैट.
```
though it's a valid one, because there are no capital letters in Hindi language.
